### DISCUSSION TITLE
Repository not found

### DISCUSSION THREAD

**Discussion by scbritton** [Comfy-Org/ComfyUI-Manager#1062] [General]
Using git-bash:

$ git clone https://github.com/ltrdata/ComfyUI-Manager.git
Cloning into 'ComfyUI-Manager'...
remote: Repository not found.
fatal: repository 'https://github.com/ltrdata/ComfyUI-Manager.git/' not found

Using git-cli:

F:\ComfyUI\ComfyUI\custom_nodes>gh repo clone ltrdata/ComfyUI-Manager
GraphQL: Could not resolve to a Repository with the name 'ltrdata/ComfyUI-Manager'. (repository)

Eventually gave up and downloaded the zip file.

**Comment by ltdrdata** [comment]
There is a typo.
`ltdrdata` is correct.

**Reply by scbritton** [reply]
Hm.  Let me look at a mirror.  Yup, I thought so - red-faced.  I feel like such an idiot now.  (Note - I'm NOT an idiot, but it's certainly possible to feel like one now and again.)
